intermediate-python-testing.md

```diff
@@ -363,6 +363,29 @@ mock_handle = mock.patch('path.to.CLASS_YOU_ARE_MOCKING')
 mock_handle.return_value = mocked_class
 ```
 
+#### `patch()` ####
```

Comment:
Can you add a subitem to the top navigation for this section?

Reply:
Actually, it might be nice to move this up, so the mock section is structured like:

```
Mock

    1. What is mock? (new heading)
        mock is a unittest submodule that allows you to remove external dependencies from your test suite... Tantalizing, right?

    2. Getting started (Your section)
        A. patch()
        B. Useful arguments
        C. Lessons learned
            Here are a few of our own hard-won lessons from early mock use... here are a few of our own hard-won lessons from early mock use.
        D. Next steps
            You can also use mock to raise exceptions to test error handling... For more on how to use mock, see the quickstart in the mock documention, as well as this excellent tutorial.

    3. requests-mock
```

Are you comfortable doing this reorg?